.github/workflows/build.yml

```diff
@@ -1387,3 +1387,33 @@ jobs:
 #          popd
 #          emcmake cmake . -DCMAKE_BUILD_TYPE=${{ matrix.build }}
 #          make
+
+  openEuler-latest-cmake-cann:
```

Comment:
openEuler-latest-cmake-cann-build, add build to job name to show this is a build test.